Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -317,6 +325,23 @@ class NetworkResponseHandler {
         }
     }
 
+    /// If handle is of type "locationHint:result", persist it to the data store
+    /// - Parameter handle: current `EventHandle` to process
+    private func handleLocationHintHandle(handle: EdgeEventHandle) {
+        guard let type = handle.type, EdgeConstants.JsonKeys.Response.EVENT_HANDLE_TYPE_LOCATION_HINT == type else { return }
+        guard let payload: [[String: Any]] = handle.payload else { return }
+
+        for locationHint in payload {
+            if let scope = locationHint[EdgeConstants.JsonKeys.Response.LocationHint.SCOPE] as? String, scope == "EdgeNetwork" {
+                guard let hint = locationHint[EdgeConstants.JsonKeys.Response.LocationHint.HINT] as? String, !hint.isEmpty else { continue }
```

Comment:
Do we have multiple hint and ttl values for EdgeNetwork scope? If not then should we break the loop instead of continue?